### PR TITLE
Support loading domain bridge settings from multiple yaml files

### PR DESCRIPTION
**Pull request by ivanpauno**

### CODE REVIEW

**Comment by ivanpauno** [comment]
I have just generated conflicts after merging https://github.com/ros2/domain_bridge/pull/63, I will resolve them ...

File: include/domain_bridge/parse_domain_bridge_yaml_config.hpp [hunk 1]
```diff
@@ -102,6 +103,29 @@ class YamlParsingError : public std::runtime_error
...
 
+/// Create a DomainBridgeConfig object from some YAML files.
+/**
+ * If the passed files have overlapping settings, the one specified in a later file will apply.
+ *
+ * See \ref parse_domain_bridge_yaml_config() for more details.
+ */
+DOMAIN_BRIDGE_PUBLIC
+DomainBridgeConfig
+parse_domain_bridge_yaml_configs(std::vector<std::filesystem::path> file_paths);
```
Comment (jacobperron) [inline include/domain_bridge/parse_domain_bridge_yaml_config.hpp:114]:
```suggestion
parse_domain_bridge_yaml_configs(const std::vector<std::filesystem::path> & file_paths);
```

**Review by jacobperron — APPROVED** [review]
LGTM!

One minor suggestion; take it or leave it.

File: include/domain_bridge/parse_domain_bridge_yaml_config.hpp (hunk 1 above)
Comment (ivanpauno) [reply include/domain_bridge/parse_domain_bridge_yaml_config.hpp]: Thanks, I was planning to make that change but I forgot.
Fixed in https://github.com/ros2/domain_bridge/pull/64/commits/7c94e5e12de969d25c287f849ec61417216b26cc.

**Review by jacobperron — APPROVED** [review]

**Pull request merged**